tests/unit/lazyfree.tcl

```diff
@@ -176,11 +176,19 @@ start_server {tags {"lazyfree"}} {
     }
 
     test "Unblocks client blocked on lazyfree via REPLICAOF command" {
+        r config resetstat
         set rd [redis_deferring_client]
 
         populate 50000 ;# Just to make flushdb async slower
         $rd flushdb
-        wait_for_blocked_client
+
+        # Verify flushdb run as lazyfree
+        wait_for_condition 50 100 {
+            [s lazyfree_pending_objects] > 0
+        } else {
+            fail "FLUSHDB didn't run as lazyfree"
+        }
+
```

Comment:
It means that depending on timing, we may not be able to test what we want to test right? e.g. if flushdb finishes before we send slaveof. Just asking to understand, otherwise it is okay as I don't see any better alternative.

Reply:
yes, unless we make the lazy free thread sleep for a while, which would be overkill.